frontend/src/app/projects/project-wrapper/project-wrapper.component.ts

```diff
@@ -33,60 +32,25 @@ export class ProjectWrapperComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit(): void {
-    const paramSubject = connectable<string>(
-      this.route.params.pipe(map((params) => params.project)),
-      {
-        connector: () => new Subject(),
-        resetOnDisconnect: false,
-      }
-    );
-
-    this.projectSubscription = paramSubject
-      .pipe(
-        switchMap((projectSlug: string) =>
-          this.projectService.getProjectBySlug(projectSlug)
-        )
-      )
-      .subscribe({
-        next: (project: Project) => {
-          this.breadcrumbsService.updatePlaceholder({ project });
-          this.projectService._project.next(project);
-        },
-        error: () => {
-          this.projectService._project.next(undefined);
-        },
-      });
-
-    this.modelsSubscription = paramSubject
-      .pipe(
-        switchMap((projectSlug: string) =>
-          this.modelService.getModels(projectSlug)
-        )
-      )
-      .subscribe({
-        next: (models: Model[]) => this.modelService._models.next(models),
-        error: () => {
-          this.modelService._models.next(undefined);
-        },
+    this.route.params
+      .pipe(map((params) => params.project))
+      .subscribe((projectSlug: string) => {
+        this.projectService.loadProjectBySlug(projectSlug);
+        this.modelService.loadModels(projectSlug);
+        this.projectUserService.getOwnProjectUser(projectSlug).subscribe();
       });
 
-    this.projectUserSubscription = paramSubject
-      .pipe(
-        switchMap((projectSlug: string) =>
-          this.projectUserService.getOwnProjectUser(projectSlug)
-        )
-      )
-      .subscribe();
-
-    paramSubject.connect();
+    this.projectService.project
+      .pipe(untilDestroyed(this))
+      .subscribe((project) =>
+        this.breadcrumbsService.updatePlaceholder({ project })
+      );
   }
 
   ngOnDestroy(): void {
-    this.projectSubscription?.unsubscribe();
-    this.modelsSubscription?.unsubscribe();
-    this.projectUserSubscription?.unsubscribe();
```

Comment:
The `projectUserSubscription`, `modelsSubscription` and `projectSubscription` are still defined, but don't seem to be used.

Reply:
I searched for all usages of subscriptions and changed all of them to use untilDestroyed(this)`, so also removed these unused subscriptions here.